scenarioo-client/app/pipes/filterArray.pipe.ts

```diff
@@ -1,11 +1,11 @@
 import { Injectable, Pipe, PipeTransform } from '@angular/core';
 
 @Pipe({
-    name: 'filter',
+    name: 'scFilterArray',
 })
 
 @Injectable()
-export class FilterPipe implements PipeTransform {
+export class FilterArrayPipe implements PipeTransform {
```

Comment:
I think it would be great to have a short documentation as a comment on each important component or pipe about what the intention of this pipe is - what is its responsibility (see also RDD Responsibility Driven Design) - Also I think the name is not very intuitive - it is to filter by a search term, right? So maybe something like "SearchFilterPipe" would be a better name?

Reply:
done together